language/functional_tests/tests/testsuite/libra_account/delegate_rotation_capability.mvir

```diff
@@ -0,0 +1,115 @@
+//! account: alice
+//! account: bob
+
+//! sender: alice
+// A module that lets the account owner rotate their auth key as they normally would, but also
+// allows a different address `master_key_address` to rotate the auth key. This is useful for
+// implementing (e.g.) a warm wallet with a cold recovery key.
```

Comment:
Better use the word hot instead of warm `(e.g., a hot wallet with a cold recovery key)`.